sci-biology/foldingathome/foldingathome-8.4.5.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+inherit python-any-r1 scons-utils systemd
+
+DESCRIPTION="Distributed computing project for simulating protein dynamics."
+HOMEPAGE="https://foldingathome.org"
+SRC_URI="
+	https://github.com/FoldingAtHome/fah-client-bastet/archive/refs/tags/v$PV.tar.gz
+		-> foldingathome-client-v$PV.tar.gz
+	https://github.com/CauldronDevelopmentLLC/cbang/archive/refs/tags/bastet-v$PV.tar.gz
+		-> foldingathome-cbang-v$PV.tar.gz
+"
+
+S="${WORKDIR}/fah-client-bastet-$PV"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="video_cards_amdgpu"
+
+DEPEND="
+	acct-group/foldingathome
+	acct-group/video
+	acct-user/foldingathome
```

Comment:
Are these really DEPENDs, I suspect we only need them at runtime.

Reply:
Okay I moved `acct-group/video` to RDPEND